cli_reference/basic_cli_operations.adoc

```diff
@@ -1,4 +1,4 @@
-= CLI Operations
+= Basic CLI Operations
```

Comment:
Agreed. The naming was inconsistent (between nav title, topic title, headings, file name) and kinda strange. I think we need to define the scope of this topic: Touch briefly on various common types of operations in each subsection (projects, builds, deployments, etc) and link to the dedicated topics for detailed info/usage? Be a comprehensive reference for _all_ CLI operations? 

For now, I'll synced up the topic title and nav bar to "CLI Operations". Leaving the file name as `basic_cli_operations` for now (don't want to break any more links), but that may need to be synced up in the future as well if something better overall is landed on.